apps/vanilla/src/pages/blackjack-by-dice/scripts/PlayerResultsDisplay.ts

```diff
@@ -0,0 +1,24 @@
+import { Subscriber } from './Interfaces';
+import { Player } from './Player';
+
+/** Display results of the player's dice roll to GUI. */
+export class PlayerResultsDisplay implements Subscriber<Player> {
+	public constructor(private element: HTMLElement) {
+	}
+
+	/**
+	 * Display results.
+	 * @param context - Object with current player index and dice number.
+	 */
+	public update(context: Player): void {
```

Comment:
```suggestion
	/** @inheritdoc */
	public update(context: Player): void {
```

You can use `@inheritdoc` if you implement an interface method. In doing so, `@inheritdoc` inherits the entire description from the interface method.